### PR TITLE
refactor: drop Cassandra specifics from plugins

### PR DESCRIPTION
**Pull request by hbagdi**
### Summary

This patch is part of a series of patches removing Cassandra support from Kong. This patch removes all Cassandra specifics from bundled plugins.

### Checklist

- [x] The Pull Request has tests
- [x] There's an entry in the CHANGELOG - will be done once the patch series is complete
- [x] There is a user-facing docs PR

### Notes for reviewer

Please rebase and merge to preserve the commit history.

### Issue reference

KAG-87

### CODE REVIEW

**Comment by hbagdi** [comment]
Added do not merge label, this PR needs to be rebased on top of master branch once #10832 is merged in.

**Comment by ADD-SP** [comment]
Rebased, and then re-test this PR.

**Review by ADD-SP — APPROVED** [review]
LGTM, but I think we need another reviewer for double-checking.

**Comment by hbagdi** [comment]
@flrgh Could you please review and merge this?

File: spec/03-plugins/09-key-auth/01-api_spec.lua [hunk 1]
```diff
@@ -469,10 +469,7 @@ for _, strategy in helpers.each_strategy() do
           assert.equal(3, #json_2.data)
 
           assert.not_same(json_1.data, json_2.data)
-          -- Disabled: on Cassandra, the last page still returns a
-          -- next_page token, and thus, an offset proprty in the
-          -- response of the Admin API.
-          --assert.is_nil(json_2.offset) -- last page
+          assert.is_nil(json_2.offset) -- last page
```
Comment (flrgh) [inline spec/03-plugins/09-key-auth/01-api_spec.lua:472]: :trophy:

**Review by flrgh — APPROVED** [review]

**Pull request merged**